.github/workflows/icon-review.yml

```diff
@@ -38,9 +38,10 @@ jobs:
       - name: Review SVG files 🔍
         run: |
           cd main
-          svgFiles=$(git diff --no-index ../main ../fork --diff-filter=ACMRTUX --name-only | grep '.svg$')
-          echo "SVG files changed: ${svgFiles}"
-          npx svg-icon-review --bigIcon ${svgFiles}
+          files=$(git diff --no-index ../main ../fork --diff-filter=ACMRTUX --name-only | grep '.svg$')
```

Comment:
We also need to count the number of files using bash's built-in methods and pass that number to a separate variable `files_count`

Reply:
Maybe I misinterpreted this code, I'm not great with the UNIX toolchain. Isn't this counting the number of files? I assumed it did becuase it was echoing the number changed, but maybe it was echoing a list of changed files?

Reply:
Oh, yeah. You're correct. It is 🤦‍♂️
Should've read it closer 😆